spec/PushController.spec.js

```diff
@@ -1063,7 +1059,11 @@ describe('PushController', () => {
         return query.find({ useMasterKey: true }).then(parseResults);
       });
     }).then(() => {
-      return pushController.sendPush(payload, where, config, auth)
+      var body = {data: {
+        alert: 'hello',
+        audience_id: audienceId
```

Comment:
This would be into the body, not in data (topLevel) as data is sent to the user ;)